### PR TITLE
initial support for ghc-9.12.1

### PR DESCRIPTION
**Pull request by shayne-fletcher**
the ghc-9.12 backport branch has been cut, this PR adds initial support for ghc-9.12.1.

### CODE REVIEW

File: ghc-lib-gen/src/GhclibgenFlavor.hs [hunk 1]
```diff
@@ -77,6 +79,7 @@ ghcSeries = \case
   f | Ghc941 <= f && f < Ghc961 -> GHC_9_4
   f | Ghc961 <= f && f < Ghc981 -> GHC_9_6
   f | Ghc981 <= f && f < Ghc9101 -> GHC_9_8
-  f | Ghc9101 <= f && f < GhcMaster -> GHC_9_10
-  GhcMaster -> GHC_9_12
+  f | Ghc9101 <= f && f < Ghc9121 -> GHC_9_10
+  f | Ghc9121 <= f && f < GhcMaster -> GHC_9_14
```
Comment (dylant-da) [inline ghc-lib-gen/src/GhclibgenFlavor.hs:83]: Did you mean to write `GHC_9_12` here? I would assume so judging by the pattern?

**Review by dylant-da — APPROVED** [review]
LGTM! Thanks for getting this through - are you still able to access our shared Slack channel?

**Comment by shayne-fletcher** [comment]
> LGTM, pending the question below. Thanks for getting this through - are you still able to access our shared Slack channel?

no, i don't seem to have access to any da slack channels anymore

**Pull request merged**